src/_constants/index.js

```diff
@@ -14,6 +14,9 @@ const CONSTANTS = KeyMirror({
   CREATE_ASSET_FAIL: true,
   LOAD_FEEDBACK_SUCCESS: true,
   LOAD_FEEDBACK_FAILURE: true,
+  CREATE_MODEL_NUMBER_SUCCESS: true,
+  CREATE_MODEL_NUMBER_FAILURE: true,
```

Comment:
Seems `prettier` isn't working properly. It should have removed this trailing comma. I'm going to look more into it so for the meantime, remove the trailing comma and empty line 19

Reply:
Hmm..if it doesn't do its work then we can remove it and just use `eslint` and `stylelint` for now. There's no need to spend so much time on a chore task, when eslint is already working. Please do not remove the precommit hook as it will still be used with the other linters
 
@HawiCaesar can you again make a quick PR that does the following:
1. removes prettier
2. updates `"react/forbid-prop-types": ['error', { forbid: ['any', 'object'] }]`  to `"react/forbid-prop-types": ["error", { "forbid": ["any", "object"] }]` in `.eslintrc`. Note the use of single quote in the first version
3. Disables `import/extensions`
4. Disables `function-paren-newline`

This will take about 10 minutes to do

Reply:
@sachungo Alright. Working on it.

Reply:
@sachungo If I remove `prettier` that means this precommit
```json
"*.{js,jsx,json,css}": [
        "./node_modules/.bin/prettier --write",
        "git add"
      ]
```
will not work and will give out an error. Shouldn't I remove the above precommit hook instead ? Please clarify ?

Reply:
Apologies for the ambiguity. I meant the eslint and stylelint precommit hooks. The prettier one is to be deleted